File: src/app/dto/ClienteCadastro/EnderecoEntregaDTOClienteCadastro.ts
export class EnderecoEntregaDtoClienteCadastro {
    EndEtg_endereco: string;
    EndEtg_endereco_numero: string;
    EndEtg_endereco_complemento: string;
    EndEtg_bairro: string;
    EndEtg_cidade: string;
    EndEtg_uf: string;
    EndEtg_cep: string;
    //codigo da justificativa, preenchdio quando está criando (do spa para a api)
    EndEtg_cod_justificativa: string;
    //descrição da justificativa, preenchdio para mostrar (da api para o spa)
    EndEtg_descricao_justificativa: string;
    OutroEndereco:boolean;
}

File: Loja/Loja.Bll/Dto/ClienteDto/EnderecoEntregaDtoClienteCadastro.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Loja.Bll.Dto.ClienteDto
{
    public class EnderecoEntregaDtoClienteCadastro
    {
        public string EndEtg_endereco { get; set; }
        public string EndEtg_endereco_numero { get; set; }
        public string EndEtg_endereco_complemento { get; set; }
        public string EndEtg_bairro { get; set; }
        public string EndEtg_cidade { get; set; }
        public string EndEtg_uf { get; set; }
        public string EndEtg_cep { get; set; }
        //codigo da justificativa, preenchdio quando está criando (do spa para a api)
        public string EndEtg_cod_justificativa { get; set; }
        //descrição da justificativa, preenchdio para mostrar (da api para o spa)
        public string EndEtg_descricao_justificativa { get; set; }
        //se foi selecionado um endereco diferente para a entrega (do spa para a api)
        public bool OutroEndereco { get; set; }
    }
}
